libs/design-system/src/lib/Pages/ToolsPage/index.tsx

```diff
@@ -111,7 +123,7 @@ const ToolsPage = ({
           </div>
         </PageSection>
 
-        <PageSection>
+        <PageSection className="container">
```

Comment:
The container has to go inside the PageSection so the borders extend all the way to the edges